honeybread-mgmtadmin/src/test/java/com/whatsub/honeybread/mgmtadmin/domain/userstorefavorite/UserStoreFavoriteControllerTest.java

```diff
@@ -1,76 +1,162 @@
 import org.springframework.test.web.servlet.ResultActions;
 
+import java.util.Collection;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.LongStream;
+
 import static org.mockito.ArgumentMatchers.anyLong;
 import static org.mockito.BDDMockito.given;
 import static org.mockito.Mockito.verify;
-import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
 import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 
+@TestConstructor(autowireMode = TestConstructor.AutowireMode.ALL)
 @WebMvcTest(UserStoreFavoriteController.class)
+@RequiredArgsConstructor
 class UserStoreFavoriteControllerTest {
     static final String BASE_URL = "/stores/favorites";
+    static final String STORE_ID_URL = BASE_URL + "/" + "store-id";
     static final String CREATE_URL = BASE_URL + "/" + 1L;
+    static final String DELETE_URL = BASE_URL + "/" + 1L;
+
+    private final MockMvc mockMvc;
 
-    @Autowired
-    MockMvc mockMvc;
+    @MockBean
+    UserStoreFavoriteService service;
 
     @MockBean
-    UserStoreFavoriteService userStoreFavoriteService;
+    UserStoreFavoriteQueryService queryService;
+
+    @Test
+    void 목록_조회_성공() throws Exception {
+        // given
+        final int 찜_개수 = 5;
+        final int 스토어_아이디_시작값 = 2;
+        Collection<Long> 찜한_스토어_아이디_목록 = 찜한_스토어_아이디_목록(스토어_아이디_시작값, 찜_개수);
+        given(queryService.getStoresByUserId(anyLong()))
+                .willReturn(new StoreIdsResponse(찜한_스토어_아이디_목록));
+
+        // when
+        ResultActions 조회_결과 = mockMvc.perform(
+                get(STORE_ID_URL)
+                        .accept(MediaType.APPLICATION_JSON)
+                        .contentType(MediaType.APPLICATION_JSON)
+        ).andDo(print());
+
+        // then
+        verify(queryService).getStoresByUserId(anyLong());
+
+        조회_결과.andExpect(status().isOk())
+                .andExpect(jsonPath("$.storeIds").isNotEmpty())
+                .andExpect(jsonPath("$.storeIds.length()").value(찜한_스토어_아이디_목록.size()))
+                .andExpect(jsonPath("$.storeIds[0]").value(2))
+                .andExpect(jsonPath("$.storeIds[1]").value(3))
+                .andExpect(jsonPath("$.storeIds[2]").value(4))
+                .andExpect(jsonPath("$.storeIds[3]").value(5))
+                .andExpect(jsonPath("$.storeIds[4]").value(6))
+                .andExpect(jsonPath("$.storeIds[5]").doesNotExist())
+        ;
+    }
+
+    private List<Long> 찜한_스토어_아이디_목록(final int startStoreId, final int size) {
+        return LongStream.range(startStoreId, startStoreId + size)
+                .mapToObj(m -> m)
+                .collect(Collectors.toList());
```

Comment:
찜한 스토어 아이디 목록을 만드는 목적이라면 다음과 같이 하는것도 좋을것 같아요
```suggestion
        return LongStream.range(startStoreId, startStoreId + size)
                .boxed()
                .collect(Collectors.toList());
```